lib/daru/view.rb

```diff
@@ -68,6 +68,17 @@ def table_library=(lib)
         end
       end
 
+      # @param libraries [Array] Adapters whose JS files will be loaded
+      # @return [void] load the dependent JS files for the adapter
+      # @example
+      #   To load the dependent JS file for Nyaplot and GoogleCharts libraries
+      #   Daru::View.load_libs_in_iruby(['nyaplot','googlecharts'])
+      def load_libs_in_iruby(libraries=[])
```

Comment:
I think we can use the `dependent_script` method to load js in iruby as well as web frameworks.
We can check for IRuby and accordingly load js .